lib/fastlane/plugin/aws_s3/actions/aws_s3_action.rb

```diff
@@ -684,6 +689,11 @@ def self.available_options
                                        is_string: true,
                                        optional: true,
                                        default_value: nil),
+          FastlaneCore::ConfigItem.new(key: :obtain_path_from_gym,
```

Comment:
Change we 💣 this in favor of just changing the description on `:xcarchive` to say... 
```
.xcarchive file for the build - set to an empty string (`""`) to not use `Actions.lane_context[SharedValues::XCODEBUILD_ARCHIVE])` as default value
```